mathics/builtin/__init__.py

```diff
@@ -37,6 +38,42 @@
 )
 
 
+def contributing_builtin_var(module, name) -> Optional[type]:
+    """
+    Returns getattr(module, name) if it is a Builtin that
+    should contribute with a definition.
+    Otherwise, returns None.
+    """
+    if name.startswith("_"):
+        return None
+
+    var = getattr(module, name)
+
+    if not (isinstance(var, type) and hasattr(var, "__module__")):
+        return None
+
+    # Skip those builtins defined in another module
+    if var.__module__ != module.__name__:
+        return None
+
+    # Skip those builtins in the module mathics.builtin.base
```

Comment:
Isn't this test covered by the one below?

Reply:
remember, var is a `class` defined in a `module`, not a module.